mobile/android/app/src/main/res/xml/network_security_config.xml

```diff
@@ -0,0 +1,14 @@
+<network-security-config>
+    <base-config>
+        <trust-anchors>
+            <certificates src="system" />
+            <certificates src="user" />
+        </trust-anchors>
+    </base-config>
+    <domain-config cleartextTrafficPermitted="false">
```

Comment:
Seems like this might have broken self hosted without SSL. https://github.com/ente-io/ente/issues/6186

Reply:
See #6201 - it _should_ fix the issue. Sorry about that!